readme.md

```diff
@@ -81,15 +82,17 @@ Once installation is working, take a look at the existing code to make sure you
 * **Question:** This code is currently _very_ insecure. Why?
 
 * **Your Answer:**
+the body is not encrypted
 
 * **Question:** What would happen if three different users tried to sign up with the same username? How can we prevent that?
 
 * **Your Answer:**
+check to confirm userid doesn't already exit in the db
```

Comment:
currently all three users could create accounts with identical usernames and passwords.